drk/.htaccess

```diff
@@ -0,0 +1,50 @@
+Header set Access-Control-Allow-Origin *
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType text/n3 .ttl
+AddType application/n-triples .nt
+AddType application/ld+json .json
+
+Options +FollowSymLinks
+
+RewriteEngine on
+
+# HTML: Visit documentation (normal)
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\*
+RewriteRule ^$           http://example.org/docs.html [L,NE,R=303]
+
+# HTML: Visit documentation (with class at the end -> make browser jump to position)
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\*
+RewriteRule ^/(.*)$     http://example.org/docs.html#$1 [L,NE,R=303]
+
+# TURTLE: Access TTL ontology
+RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.*
+RewriteRule ^$           http://example.org/ontology.ttl [L,R=303]
+
+RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.*
+RewriteRule ^/(.*)       http://example.org/ontology.ttl [L,R=303]
+
+# No matching content type: Visit documentation
+RewriteRule ^/(.*)      http://example.org/docs.html#$1 [L,NE,R=303]
+
+# DRK ontology resources (to be updated later)
+# DRK ontology and its documentation in human-readable form
+RewriteRule ^drk$ https://w3id.org/drk/ [R=302,L]
```

Comment:
RewriteRules are relative.  This makes to https://w3id.org/drk/drk.  And redirecting to /drk/ doesn't make a lot of sense.  The below rules are redirecting to root files that will never exist.  Please update these, fix the above example rules and test.